mitodl/ocw-studio
# added migration to remove duplicate fields and correct value

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/4372

### Description (What does it do?)
The migration aims to remove redundant field `has_external_licence_warning ` and add updated field `has_external_license_warning` for external resources. The value however will be computed based on the `external_url` field of resource.

### How can this be tested?
1. Create a new course or use an existing one.
2. Add an external resource in the course from `ocw-studio`.
3. Go to `admin` panel and find the external resource under `Website Content`: `http://localhost:8043/admin/websites/websitecontent/`
4. in the metadata field for the content, add `has_external_licence_warning: true` and save.
5. In `ocw-studio` repo, switch to branch `umar/4372-ocw-remove-duplicate-metadata-field-for-external-resources`
6. Apply migration: `docker-compose exec web ./manage.py migrate websites`
7. Check the metadata of the resource from admin panel. It should remove `has_external_licence_warning` field and add or update `has_external_license_warning` field.
